test/modules/vass/use-a-Math-function.prediff

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+output=$2
+sed -e 's/:[0-9][0-9][0-9]*:/::/' $output > $output.tmp
+mv $output.tmp $output
```

Comment:
Is this actually necessary?  I thought our handling of .bad files already removed variability in provided module locations

Reply:
Cool, I removed them in #14541.